src/app/campaign/flight/flight-form.component.html

```diff
@@ -69,6 +69,9 @@ <h2>Zones</h2>
         [flightZones]="flight?.zones"
         [isCompanion]="flight?.isCompanion"
         [zoneOptions]="zoneOptions"
```

Comment:
I don't love passing these all the way down, might just decide to use the store and selectors at the child component level (as long as we don't get too wild with it. it's totally fine, just intentionally deciding where the boundaries are keeps things following the same patterns.)